src/preprocessors.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument,@typescript-eslint/no-unsafe-assignment */
```

Comment:
i'll just disable the `no-unsafe-*` family in the eslint config and bump the version, they've ended up being a lot of friction for very little value :sweat_smile: